examples/pytorch/graphsage/cugraph_sampler.py

```diff
@@ -0,0 +1,119 @@
+"""
```

Comment:
Need Nvidia copyright block at top

Reply:
Because the integration is using graphsage as example, so it is in the example folder